www/src/data/sidebars/doc-links.yaml

```diff
@@ -144,21 +144,21 @@
     - title: Styling your site
       link: /docs/styling/
       items:
-        - title: Layout Components
+        - title: Using layout components
```

Comment:
I would title case this and "Using Component CSS", otherwise this looks great! Love the active verb changes.